app.js

```diff
@@ -0,0 +1,203 @@
+let books = [];
+
+function Book(title, status, author, imageUrl, numPages) {
+  this.title = title;
+  this.status = status;
+  this.author = author;
+  this.imageUrl = imageUrl;
```

Comment:
- [ ] [OPTIONAL] You could assign a default image at the declaration of this function so that even when a user doesn't submit one something will still show up:

` function Book(title, status, author, imageUrl='https://mydefaultimgUrl.com', numPages) { `

Reply:
You are right will do that right now